### PR TITLE
Fixes redis and pg

### PR DESCRIPTION
**Pull request by rilian**

### CODE REVIEW

**Comment by tilast** [comment]
ok for me :+1:

**Comment by DenisUA** [comment]
looks good

File: install.rb [hunk 1]
```diff
@@ -1,3 +1,5 @@
+require 'yaml'
```
Comment (ololyay) [inline install.rb]: for what?

Comment (rilian) [inline install.rb]: cause for some reason YAML is not working here out of the box

File: packages/redis.rb [hunk 2]
```diff
@@ -15,16 +21,11 @@
...
-  file config_file_path, contents: File.read(config_template), sudo: true
-
-  runner "chmod 755 #{config_file_path}"
+  runner "sudo sed -i 's/# bind 127.0.0.1/bind 127.0.0.1/g' /etc/redis/redis.conf"
   runner 'service redis-server restart'
 
   verify do
-    has_file config_file_path
-    has_permission config_file_path, '755'
+    # Find that config file does not have commented bind param
```
Comment (sudodoki) [inline packages/redis.rb]: > Find
it actually `verify`, no?

**Pull request merged**